src/RadixDlt.NetworkGateway.PostgresIntegration/Services/EntityStateQuerier.cs

```diff
@@ -676,7 +685,7 @@ LIMIT 1
             },
             cancellationToken: token);
 
-        var result = await _dbContext.Database.GetDbConnection().QueryAsync<NonFungibleIdLocationViewModel>(cd);
+        var result = await _dapperWrapper.QueryAsync<NonFungibleIdLocationViewModel>(_dbContext.Database.GetDbConnection(), cd);
```

Comment:
Having this fully-blown DapperWrapper shouldn't we pass db connection as one of its constructor arguments? Just thinking...

Reply:
hmm sounds good, i'll have to just switch it to transient dependency and inject dbconnection in DI, but that's easy.